src/main/java/net/dongliu/apk/parser/struct/resource/ResTableConfig.java

```diff
@@ -58,12 +59,24 @@ public class ResTableConfig {
     private short screenLayout;
     // uint8_t
     private short uiMode;
-    // uint8_t
-    private short screenConfigPad1;
-    // uint8_t
-    private short screenConfigPad2;
+    // uint16_t
+    private int smallestScreenWidthDp;
     //uint32_t screenConfig;
 
+    // uint16_t
+    private int screenWidthDp;
+    // uint16_t
+    private int screenHeightDp;
+    //uint32_t screenSizeDp;
+
+    // uint8_t. Contains round/notround qualifier.
+    private short screenLayout2;
+    // uint8_t. Wide-gamut, HDR, etc.
+    private short colorMode;
+    // uint16_t. Reserved padding.
+    private int screenConfigPad2;
```

Comment:
Looks like this was a short before.

Reply:
It was incorrectly defined in the original repo, but not causing any issue because the parser wasn't using it